test/parallel/test-http-client-aborted-event.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+require('../common');
+var http = require('http');
+var assert = require('assert');
+
+var server = http.Server(function(req, res) {
+  console.log('Server accepted request.');
+  res.write('Part of my res.');
+
+  res.destroy();
+});
+
+var responseAborted = false;
+
+server.listen(0, function() {
+  http.get({
+    port: this.address().port,
+    headers: { connection: 'keep-alive' }
+  }, function(res) {
+    server.close();
+
+    console.log('Got res: ' + res.statusCode);
+
+    res.on('aborted', function() {
+      console.log('Response aborted.');
+      responseAborted = true;
+    });
+  });
+});
+
+
+process.on('exit', function() {
+  assert.ok(responseAborted);
+});
```

Comment:
I think `responseAborted` can be omitted when using `common.mustCall` everywhere